examples/express/hold_app.py

```diff
@@ -25,4 +25,4 @@ def txt():
 
 ui.input_slider("n", "N", 1, 100, 50)
 
-shiny.ui.output_code("txt", inline=True)
```

Comment:
Thank you! 

Reference: https://shiny.posit.co/py/api/core/ui.output_code.html

No `inline` argument